earth_enterprise/src/common/sharedstring_unittest.cpp

```diff
@@ -1,118 +0,0 @@
-// Copyright 2019 The Open GEE Contributors
```

Comment:
It wouldn't hurt to have some very basic unit tests for AssetRefKey since it does involve some code.